<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>523</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>523</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>523</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>A-Maze-ing!</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(210,210,220);
}

QPushButton {
background-color: rgb(169,63,85);
color: rgb(255,255,255);
border-radius: 10px;
}

QPushButton:pressed {
background-color: rgb(127, 46, 62);
}

QPushButton:disabled {
background-color: rgb(200, 200, 200);
}

QLabel {
	color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>515</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>515</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>515</height>
     </size>
    </property>
    <widget class="QWidget" name="mazeTab_"/>
    <widget class="QWidget" name="caveTab_"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA_Maze_ing">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMaze"/>
    <addaction name="actionCave"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuA_Maze_ing"/>
  </widget>
  <action name="actionMaze">
   <property name="text">
    <string>Maze</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionCave">
   <property name="text">
    <string>Cave</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
